build/cyrus/files/imapd.conf

```diff
@@ -0,0 +1,140 @@
+# Suggested minimal imapd.conf
+# See imapd.conf(5) for more information and more options
+
+# Space-separated users who have admin rights for all services.
+# NB: THIS MUST BE CONFIGURED
+admins: cyrus
+
+# Space-separated list of IPv4 addresses and netmasks from which admin logins
+# are premitted. Comment out to allow admin access from everywhere.
+# admins_ip: 127.0.0.1 10.0.0.0/8 192.168.1.0/24
+admins_ip: 127.0.0.1
+
+# The amount of server information to show to users
+serverinfo: min
+
+###################################################################
+## File, socket and DB location settings.
+###################################################################
+
+# Configuration directory
+configdirectory: /var/$(PREFIX)
+
+# Directories for proc and lock files
+proc_path: /$(RUNDIR)/proc
+mboxname_lockpath: /$(RUNDIR)/lock
+
+# Locations for DB files
+# The following DB are recreated upon initialisation, so should live in
+# ephemeral storage for best performance. /$(RUNDIR) is mounted
+# on tmpfs by svc:/network/cyrus-setup:default
+duplicate_db_path: /$(RUNDIR)/deliver.db
+ptscache_db_path:  /$(RUNDIR)/ptscache.db
+statuscache_db_path: /$(RUNDIR)/statuscache.db
+tls_sessions_db_path: /$(RUNDIR)/tls_sessions.db
+
+# Which partition to use for default mailboxes
+defaultpartition: default
+partition-default: /var/$(PREFIX)/mail
+
+# If sieveusehomedir is false (the default), this directory is searched
+# for Sieve scripts.
+sievedir: /var/$(PREFIX)/sieve
+
+###################################################################
+## Important: KEEP THESE IN SYNC WITH cyrus.conf
+###################################################################
+
+lmtpsocket: /var/$(PREFIX)/socket/lmtp
+idlesocket: /var/$(PREFIX)/socket/idle
+notifysocket: /var$(PREFIX/socket/notify
```

Comment:
I get the feeling there is a typo here shouldn't this read /var/$(PREFIX/socket/notify analog to the ones above ?

Reply:
Indeed, thanks!